test/lib/linter/snapshots/linter.ts.md

```diff
@@ -821,6 +804,23 @@ Generated by [AVA](https://avajs.dev).
         ],
         warningCount: 0,
       },
+      {
+        coverageInfo: [],
+        errorCount: 1,
+        fatalErrorCount: 0,
+        filePath: 'ui5.yaml',
```

Comment:
This one confuses me. I'll have to check what happens here. I would have expected that `ui5.yaml` comes before `webapp/`, as `u` comes before `w`.

Reply:
Solved in 9669083